lib/view/icu_non_intensivist/ventilation/ventilation_view.dart

```diff
@@ -17,24 +20,39 @@ class VentilationView extends StatelessWidget {
     // Analytics set screen name, stays until another screen changes it
     Analytics.analyticsScreen(Constants.analyticsVentilationScreen);
 
-    final tabs = ventilationGuide.map((e) => e.heading).toList();
-
-    return TabViewTemplate(
-      Strings.ventilationTitle,
-      color: AppColors.blue50,
-      tabs: tabs,
-      actions: <Widget>[
-        IconButton(
-          icon: SvgPicture.asset(
-            'assets/images/icon/SVG/icon_infographic.svg',
-          ),
-          onPressed: () =>
-              Navigator.of(context).pushNamed(Routes.ventilationInfographic),
-        )
-      ],
-      children: ventilationGuide
-          .map((e) => VentilationGuideContainer(guideSections: e.guideList))
-          .toList(),
-    );
+    final tabs = ventilationGuide.map((e) {
+      if (e is IntubationContent) {
+        return e.heading;
+      } else if (e is ICUDailyRoundSteps) {
+        return e.heading;
+      }
+      return e.toString();
+    }).toList();
+    return TabViewTemplate(Strings.ventilationTitle,
+        color: AppColors.blue50,
+        tabs: tabs,
+        indicatorColor: AppColors.blue900,
+        actions: <Widget>[
+          IconButton(
+            icon: SvgPicture.asset(
+              'assets/images/icon/SVG/icon_infographic.svg',
+            ),
+            onPressed: () =>
+                Navigator.of(context).pushNamed(Routes.ventilationInfographic),
+          )
+        ],
+        children: ventilationGuide.map((e) {
+          if (e is IntubationContent) {
+            return IntubationContentViewTemplate(
+              content: e,
+              color: AppColors.blue500,
+            );
+          } else if (e is ICUDailyRoundSteps) {
+            return ICUDailyRoundStepsContainer(
+                steps: e, backgroundColor: AppColors.blue500);
```

Comment:
trailing comma pls :slightly_smiling_face:

Reply:
Yes please to trailing commas.